### PR TITLE
Add stored data retrieval data views

### PR DESCRIPTION
**Pull request by kelseyduffy**
added stored value retrieval for data views
- breaking change for interpolated data function name

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Version History
 
+## 0.1.4-preview / 2021-08-06
```
Comment (derekendres) [inline HISTORY.md:3]: if we are breaking things i might consider it .2

Comment (kelseyduffy) [reply HISTORY.md]: I changed it to 0.2.0, and also added a note explicitly stating the function's old and new names in the history file

**Comment by kelseyduffy** [comment]
switched from a breaking change to an aliasing of the previous data function.

File: HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Version History
 
+## 0.1.4-preview / 2021-08-10
+
+- Added stored data retrieval for data views
+- Aliased DataViewClient.GetDataViewData() to DataViewClient.GetDataViewInterpolatedData()
+- updated dependencies
```
Comment (dcbroad3) [inline HISTORY.md:7]: Minor: `Updated` to use consistent casing?

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (dcbroad3) [inline package-lock.json:1]: Why are we adding a `package-lock.json` to this repo? Seems unintended?

**Review by dcbroad3 — APPROVED** [review]

**Pull request merged**